x/evm/migrations/v3/store.go

```diff
@@ -69,7 +69,7 @@ func MigrateStore(
 		EnableCall:          legacyParams.EnableCall,
 		ExtraEIPs:           legacyParams.ExtraEIPs,
 		ChainConfig:         newChainConfig,
-		EIP712AllowedMsgs:   MigrateEIP712AllowedMsgs(legacyParams.EIP712AllowedMsgs),
+		EIP712AllowedMsgs:   legacyParams.EIP712AllowedMsgs,
```

Comment:
Since the EIP712AllowedMsg data hasn't changed, we can use the same type for both legacy and new params